firmware/.gitignore

```diff
@@ -0,0 +1,5 @@
+.pio
+.vscode/.browse.c_cpp.db*
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/ipch
```

Comment:
Hey @Peguy-WANDA you seem to have this file in every PR you send 😉 Please, either delete it locally, or commit file-by-file and not all files.

Reply:
Hello Daniel, Ok it was a mistake sorry, So let me remove it